tests/test_parameter_interpolation.py

```diff
@@ -132,9 +132,10 @@ def u_reference(x, params, box, lamb):
                 print("lambda", lamb, "cutoff", cutoff, "precision", precision, "xshape", coords.shape)
 
                 args = copy.deepcopy(test_potential.args)
+                pi = f"{np.pi:.14f}"
```

Comment:
This goes from 36 decimal places to 14 decimal places, what is the intention with this change?

Reply:
The reason for removing the `#define` is that this is expanded by the preprocessor invocation when we generate the `k_lambda_transformer_jit.cui`, so isn't available for use in custom expressions (there might be some workarounds for this, but passing in the value from Python seemed the simplest solution to me).

14 decimal places is roughly the precision of IEEE 754 doubles, so I don't think we lose anything by truncating (`np.pi` is a double, so the decimals after the 15th deviate from the true value of pi anyway).

Reply:
Added a note in a2c3325d192c6962c20e70b1b641ac39c5d6d1a7